home/views.py

```diff
@@ -86,11 +89,17 @@ def dashboard(request):
             return render(request, 'home/dashboard.html', context)
 
         # Classwork/Homework info
-        cw_hw = ClassworkHomework.objects.filter(cal_date=calendar, section__section_id=query_section).first()
+        cw_hw = ClassworkHomework.objects.filter(
+            cal_date=calendar, section__section_id=query_section).first()
         context['cw_hw'] = cw_hw
 
+        # subject scheduled
+        subject_scheduled = schedule.get().get_subject_display()
```

Comment:
Why `get()`?

Reply:
SUBJECT is a tuple and by .get() i am getting its value. when i remove .get() i am getting an error "'QuerySet' object has no attribute 'get_subject_display'"). also i am not able to figure out another way to do it without .get(), i am only getting key like 'mat' and 'hin' instead of Mathematics and Hindi when i do it without .get()

Reply:
Yeah, that's because `schedule` is a QuerySet and not an instance of the `Schedule` model. 

Replace 
```
if not schedule.exists():
    return redirect('home:dashboard')
```
with
```
if schedule.exists():
    schedule = schedule[0]
else:
    return redirect('home:dashboard')
```

and then it will work fine without using `get()`.